### PR TITLE
:recycle: refactor: add default_nick_name table and get random nickname

### PR DESCRIPTION
**Pull request by seonghun-dev**
resolve : https://github.com/depromeet/street-drop-server/issues/148

### CODE REVIEW

**Review by siyeonSon — APPROVED** [review]
고생하셨습니당 👍

File: backend/streetdrop-domain/src/main/java/com/depromeet/user/DefaultNickName.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+        @Id
+        @GeneratedValue(strategy = IDENTITY)
+        private Long id;
+
+        @Column(length = 10)
+        private String preNickName;
+
+        @Column(length = 10)
+        private String postNickName;
```
Comment (yunyoung1819) [inline backend/streetdrop-domain/src/main/java/com/depromeet/user/DefaultNickName.java:23]: 저희 닉네임 최대 10글자까지 아니였나요??

**Review by yunyoung1819 — CHANGES_REQUESTED** [review]
Good!

File: backend/streetdrop-domain/src/main/java/com/depromeet/user/DefaultNickName.java (hunk 1 above)
Comment (seonghun-dev) [reply backend/streetdrop-domain/src/main/java/com/depromeet/user/DefaultNickName.java:23]: 확인해보니 10글자이군요. 수정해두겠습니다

**Pull request merged**